packages/dbos-cloud/applications/get-app-logs.ts

```diff
@@ -1,35 +1,66 @@
 import axios , { AxiosError } from "axios";
 import { handleAPIErrors, getCloudCredentials, getLogger, isCloudAPIErrorResponse, retrieveApplicationName } from "../cloudutils.js";
 
-export async function getAppLogs(host: string, last: number): Promise<number> {
+type LogResponse = {
+  end: boolean;
+  next_timestamp :string;
+  body: string;
+};
+
+
+export async function getAppLogs(host: string, last: number, pagesize:number): Promise<number> {
   if (last != undefined && (isNaN(last) || last <= 0)) {
     throw new Error('The --last parmameter must be an integer greater than 0');
   }
   if (last == undefined) {
     last = 0      //internally, 0 means "get all the logs." This is the default.
   }
+
+  if (pagesize != undefined && (isNaN(pagesize) || pagesize <= 0)) {
+    throw new Error('The --pagesize parmameter must be an integer greater than 0');
+  }
+  if (pagesize == undefined) {
+    pagesize = 1000
+  }
+
   const logger = getLogger();
   const userCredentials = getCloudCredentials();
   const bearerToken = "Bearer " + userCredentials.token;
-
   const appName = retrieveApplicationName(logger);
+  const url = `https://${host}/v1alpha1/${userCredentials.userName}/logs/applications/${appName}`;
+  const headers = {
+    "Content-Type": "application/json",
+    Authorization: bearerToken,
+  }
+  const params = {
+    last: last,
+    limit: pagesize,
+    format: 'json'
+  }
   if (!appName) {
     return 1;
   }
-  logger.info(`Retrieving logs for application: ${appName}`)
-
   try {
-    const res = await axios.get(`https://${host}/v1alpha1/${userCredentials.userName}/logs/applications/${appName}?last=${last}`, {
-      headers: {
-        "Content-Type": "application/json",
-        Authorization: bearerToken,
-      }
-    });
-    if (res.data == "") {
+    const res = await axios.get(url, {headers: headers, params: params});
+    const logResponse = res.data as LogResponse
+    if (logResponse.end && logResponse.body == "") {
       logger.info(`No logs found for the specified parameters`);
     } else {
-      logger.info(`Successfully retrieved logs of application: ${appName}`);
-      logger.info(res.data)
+      console.log(logResponse.body.trimEnd())
```

Comment:
Should this be `logger.info()`?

Reply:
I changed this to console.log because logger.info adds its own [TIME][info]... prefix. These logs have a similar prefix already. So when paginating you'd see
```
[timestamp][info]: [inner_log_timestamp][info][body]
[inner_log_timestamp][info][body]
[inner_log_timestamp][info][body]
...
[timestamp][info]: [inner_log_timestamp][info][body]
[inner_log_timestamp][info][body]
[inner_log_timestamp][info][body]
```
I decided to essentially send the inner body of the logs to stdout "exactly as is". Does that make sense?